.maintain/frame-weight-template.hbs

```diff
@@ -64,10 +64,10 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 		{{~#each benchmark.components as |c| ~}}
 		{{~#if (not c.is_used)}}_{{/if}}{{c.name}}: u32, {{/each~}}
 	) -> Weight {
-		Weight::from_ref_time({{underscore benchmark.base_weight}} as u64)
+		Weight::from_parts({{underscore benchmark.base_weight}} as u64, 0)
```

Comment:
I have updated both files based on the substrate template from v0.9.42. There is only one small difference in the way the template is used, but it may require more work in `run_benchmarks.sh`, which is significantly different from substrate. I will leave them for now so we can evaluate the output of the benchmarks.